harness/determined/estimator/_estimator_context.py

```diff
@@ -150,7 +157,7 @@ class EstimatorExperimentalContext(_data_layer.DataLayerContext):
 
     def __init__(self, env: det.EnvContext, hvd_config: horovod.HorovodContext) -> None:
         super().__init__(env=env, hvd_config=hvd_config)
-        self._allgather_fn = None  # type: Optional[Callable[[Any], List]]
+        self._allgather_fn = default_allgather_fn  # type: Callable[[Any], List]
```

Comment:
What is it to fix?

Reply:
Otherwise you get an error when you use custom reducers outside of determined.  I explained in the docstring of `default_allgather_fn`